client/src/components/AboutSection.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import "./AboutSection.css";
+import Dancers from "../images/about-dancers.png";
+
+const content = {
+	title: "Mapping Black Atlantic Rhythms",
+	firstParagraph:
+		"Do you love music or dance? Have you ever wondered about the African roots of hip-hop, jazz, blues and reggae or dancehall, gwo-ka, samba or tango? ",
+	secondParagraph:
```

Comment:
You could use an array of `paragraphs: []` but that's probably overkill. I'm fine with this.